AUTHORS.md

```diff
@@ -0,0 +1,15 @@
+<!--
```

Comment:
This is not a code, so one of the CC licenses would fit better e.g  CC0-1.0 (Public Domain)
Same for CONTRIBUTING.md, ChangeLog and README.md

Reply:
Yes, it would fit better, but 
- it was GPL-2+ before
- we would reduce the number of licenses, if we stick to GPL-2+ instead of CC0 and make it less complex
- if all contributors agree, we can re license it to the more permissive CC0. I have no objections.
- it makes sense to put these insignificant files under the major license of the project see also https://reuse.software/faq/#uncopyrightable and https://reuse.software/tutorial/#insignificant-files 

What do you think?

Reply:
Whatever you choose for me will be fine, it doesn't have to be CC0.
If you choose to change the license:
CONTRIBUTING.md  and README.md you're the autor, so it's up to you. 
ChangeLog you have to ask @ninthclowd 
AUTHORS.md Formally you're the author, but I'd ask @ninthclowd and @jmwilson

Reply:
OK I merge it for now. We can re license later, if we want to.